build_projects/dotnet-host-build/PrepareTargets.cs

```diff
@@ -244,6 +273,11 @@ public static BuildTargetResult RestorePackages(BuildTargetContext c)
                 .Execute()
                 .EnsureSuccessful();
 
+            dotnet.Restore("--verbosity", "verbose", "--disable-parallel")
+                .WorkingDirectory(Path.Combine(c.BuildContext.BuildDirectory, "tools", "dotnet-deb-tool"))
```

Comment:
seems like it should work... `dotnet restore` should go through all the dirs, right? and process each one as if it was called directly on it.